catalogs/catalog-lakehouse-paimon/src/test/java/org/apache/gravitino/catalog/lakehouse/paimon/utils/TestTableOpsUtils.java

```diff
@@ -0,0 +1,290 @@
+import static org.apache.gravitino.rel.TableChange.updateComment;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrowsExactly;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.function.Consumer;
+import org.apache.commons.lang3.tuple.Pair;
+import org.apache.gravitino.rel.TableChange;
+import org.apache.gravitino.rel.expressions.literals.Literals;
+import org.apache.gravitino.rel.indexes.Index.IndexType;
+import org.apache.gravitino.rel.types.Types;
+import org.apache.gravitino.rel.types.Types.DoubleType;
+import org.apache.gravitino.rel.types.Types.FloatType;
+import org.apache.gravitino.rel.types.Types.IntegerType;
+import org.apache.gravitino.rel.types.Types.ListType;
+import org.apache.gravitino.rel.types.Types.MapType;
+import org.apache.gravitino.rel.types.Types.StringType;
+import org.apache.paimon.schema.SchemaChange;
+import org.apache.paimon.schema.SchemaChange.AddColumn;
+import org.apache.paimon.schema.SchemaChange.DropColumn;
+import org.apache.paimon.schema.SchemaChange.Move.MoveType;
+import org.apache.paimon.schema.SchemaChange.RemoveOption;
+import org.apache.paimon.schema.SchemaChange.RenameColumn;
+import org.apache.paimon.schema.SchemaChange.SetOption;
+import org.apache.paimon.schema.SchemaChange.UpdateColumnComment;
+import org.apache.paimon.schema.SchemaChange.UpdateColumnNullability;
+import org.apache.paimon.schema.SchemaChange.UpdateColumnPosition;
+import org.apache.paimon.schema.SchemaChange.UpdateColumnType;
+import org.apache.paimon.schema.SchemaChange.UpdateComment;
+import org.apache.paimon.types.DataTypeRoot;
+import org.junit.jupiter.api.Test;
+
+public class TestTableOpsUtils {
+
+  @Test
+  void testBuildSchemaChange() {
+    // Test supported table changes.
+    assertTableChange(
+        addColumn(
+            fieldName("col_1"),
+            IntegerType.get(),
+            AddColumn.class.getSimpleName(),
+            TableChange.ColumnPosition.first(),
+            false,
+            false),
+        AddColumn.class,
+        schemaChange -> {
+          AddColumn addColumn = (AddColumn) schemaChange;
+          assertEquals("col_1", addColumn.fieldName());
+          assertEquals(DataTypeRoot.INTEGER, addColumn.dataType().getTypeRoot());
+          assertEquals(AddColumn.class.getSimpleName(), addColumn.description());
+          assertNotNull(addColumn.move());
+          assertEquals(MoveType.FIRST, addColumn.move().type());
+          assertEquals("col_1", addColumn.move().fieldName());
+          assertNull(addColumn.move().referenceFieldName());
+          assertFalse(addColumn.dataType().isNullable());
+        });
+    assertTableChange(
```

Comment:
could you split the test to make it more clear?

Reply:
updated.